Doc/library/csv.rst

```diff
@@ -467,9 +467,14 @@ DictWriter objects have the following public method:
 
 .. method:: DictWriter.writeheader()
 
-   Write a row with the field names (as specified in the constructor).
+   Write a row with the field names (as specified in the constructor) to
+   the writer's file object, formatted according to the current dialect and
+   return the return value of the :meth:`csvwriter.writerow` used internally.
```

Comment:
> return the return value of the :meth:`csvwriter.writerow` used internally.

Ok but... the return value of writerow() is not documented... I would prefer to document it as well. It seems to be the result of file.writeline(line).

Reply:
Hi @vstinner! There is another issue opened for this (https://bugs.python.org/issue34713) which is what prompted me to do some research and update the patch in the bpo report in this PR. I'm waiting a few days to let @nchammas time to write a PR if he wishes so, otherwise I think it may be nice to keep the easy issue for the next mentored sprint.

Reply:
I would prefer to see the doc of the two methods updated at the same time. I'm not sure that it's an easy issue, because I'm not sure myself how to document writerow() return value.

Reply:
@vstinner, I added documentation for the `csvwriter.writerow` method.